tutorial/westeros/westeros_addon_technologies.ipynb

```diff
@@ -0,0 +1,607 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Westeros Tutorial\n",
```

Comment:
Use a title format consistent with other Westeros tutorials, and possibly numbering as you wish to bring this after a certain tutorial, like 3c.

Reply:
done.